docs/run_publish/references.md

```diff
@@ -100,6 +105,42 @@ This displays the current version.
 0.19.1
 ```
 
+### reindex
+
+- In order to use this command you need to have `@subql/node` v1.10.0 or above.
+
+When using reindex command, historical must be enabled for the targeted project(`--disable-historical=false`).
+
+Use `--targetHeight=<blockNumber>` with `reindex` to remove indexed data and reindex from specified block height.
+`-f`, `--subquery` flag must be passed in, to set path of the targeted project.
+
+::: info Note
+Once the command is executed, the application would exit upon completion.
```

Comment:
Why wouldn't it continue to index?

Reply:
on the hosted service side, when implementing reindex/force-clean its better to keep it a one off command